ui/page/root/wallet_settings_page.go

```diff
@@ -297,9 +359,10 @@ func (pg *WalletSettingsPage) pageSections(gtx C, title string, body layout.Widg
 				layout.Rigid(func(gtx C) D {
 					return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
 						layout.Rigid(func(gtx C) D {
-							txt := pg.Theme.Label(values.TextSize14, title)
+							txt := pg.Theme.Label(values.TextSize20, title)
 							txt.Color = pg.Theme.Color.GrayText2
-							return txt.Layout(gtx)
+							txt.Font.Weight = font.Bold
+							return layout.Inset{Bottom: values.MarginPadding15}.Layout(gtx, txt.Layout)
```

Comment:
Space between the title and the next widget is 24px on figma. It's 26 in some places, but most places have a spacing of 24px, so let's go with 24px.